packages/ifixit-api-client/index.tsx

```diff
@@ -60,13 +60,21 @@ export class IFixitAPIClient {
    }
 
    async fetch(endpoint: string, statName: string, init?: RequestInit) {
+      const pskToken = process.env.DEV_API_AUTH_TOKEN;
+      const authHeader: { Authorization?: string } = pskToken
+         ? { Authorization: `PSK ${pskToken}` }
+         : {};
       const url = `${this.origin}/api/${this.version}/${endpoint}`;
       const response = await timeAsync(
          `ifixit-api.${init?.method?.toLowerCase() || 'get'}.${statName}`,
          () =>
             fetch(url, {
                credentials: 'include',
                ...init,
+               headers: {
+                  ...authHeader,
+                  ...init?.headers,
```

Comment:
I don't think this is safe with a value that may be null or undefined:

![image](https://user-images.githubusercontent.com/26855/236586662-33f447eb-d4cc-4e74-92f0-5b954f892c42.png)

Huh... but the previous `...init` should have been unsafe too... why wouldn't typescript complain then?

Reply:
TIL It's safe with splatting into objects:
![image](https://user-images.githubusercontent.com/26855/236586850-d99b9073-32e7-474a-bdae-ce0a0417f566.png)